docs/auth-provider-setup/aggregate-verifier.mdx

```diff
@@ -38,25 +38,34 @@ combined. The following table shows all possible combinations of login providers
 
 :::
 
-| First Sub-Verifier | Google | Facebook | Twitch | Discord | Auth0 | Email Passwordless | SMS Passwordless |
-| :----------------- | :----: | :------: | :----: | :-----: | :---: | :----------------: | :--------------: |
-| **Google**         |   ✅   |    ❌    |   ❌   |   ❌    |  ✅   |         ✅         |        ❌        |
-| **Facebook**       |   ❌   |    ❌    |   ❌   |   ❌    |  ✅   |         ❌         |        ❌        |
-| **Twitch**         |   ❌   |    ❌    |   ✅   |   ❌    |  ❌   |         ❌         |        ❌        |
-| **Discord**        |   ❌   |    ❌    |   ❌   |   ✅    |  ❌   |         ❌         |        ❌        |
-| **Email P'less**   |   ✅   |    ❌    |   ❌   |   ❌    |  ✅   |         ✅         |        ❌        |
-| **SMS P'less**     |   ❌   |    ❌    |   ❌   |   ❌    |  ✅   |         ❌         |        ✅        |
-| **Auth0**          |   ✅   |    ✅    |   ❌   |   ❌    |  ✅   |         ✅         |        ✅        |
+| First Sub-Verifier | Google | Facebook | Twitch | Discord | Auth0\* | Email Passwordless | SMS Passwordless |
+| :----------------- | :----: | :------: | :----: | :-----: | :-----: | :----------------: | :--------------: |
+| **Google**         |   ✅   |    ❌    |   ❌   |   ❌    |   ✅    |         ✅         |        ❌        |
+| **Facebook**       |   ❌   |    ❌    |   ❌   |   ❌    |   ❌    |         ❌         |        ❌        |
```

Comment:
Let's remove Facebook from here, we are removing this from Dashboard combination UI as well.

Reply:
<img width="1155" alt="image" src="https://github.com/user-attachments/assets/2b33ee9f-e62b-4c81-b86e-ac32edf926bb" />
Resolved